### DISCUSSION TITLE
Notify additional secondary servers but only for certain domains

### DISCUSSION THREAD

**Discussion by bradley-baker** [PowerDNS/pdns#13389] [Q&A] [answered]
We have approximately 200 zones on our PowerDNS servers setup in a primary/secondary configuration. For a small number of zones (around 5) we need to send NOTIFY requests to additional secondary servers. 

My understanding is that by default PowerDNS sends NOTIFYs to the name servers but in this case, the provider doesn't listen to NOTIFYs on the name servers but on a different set of IPs. 

So for example their name servers are: 
ns100.digicertdns.com (208.94.148.63)
ns101.digicertdns.com (208.80.124.63)
ns102.digicertdns.com (208.80.126.63)

But they want us to NOTIFY the following IPs instead:
208.94.147.135
208.94.150.198
63.219.151.12

I can set the following option in the PowerDNS config:
```
also-notify=208.94.147.135,208.94.150.198, 63.219.151.12
```

But then I'm notifying them for roughly 195 domains they don't host and don't care about. 

I suppose this is relatively harmless but I feel like we're almost spamming their server with garbage NOTIFYs that their server has to ignore. 

We want to be able to do something to make also-notify but make it zone-specific. 

So something like:
also-notify-zone-list: domain1.com,domain2.com

Or better yet:
also-notify:
domain1.com:208.94.147.135,208.94.150.198, 63.219.151.12
domain2.com:208.94.147.135,208.94.150.198, 63.219.151.12

Is something like this possible with PowerDNS or do we just have to live with our server spamming their server with unnecessary NOTIFYs?

Thanks
Brad

**Comment by aj-gh** [comment, marked as answer]
Did you try setting "ALSO-NOTIFY" in the per-zone metadata? I'd assume that would fix your issue.

See [documentation](https://doc.powerdns.com/authoritative/domainmetadata.html#also-notify):
```
When notifying this domain, also notify this nameserver (can occur multiple times). The nameserver may contain an optional port number. e.g.:
pdnsutil set-meta powerdns.org ALSO-NOTIFY 192.0.2.1:5300
pdnsutil set-meta powerdns.org ALLOW-AXFR-FROM 2001:db8:53::1
```

Note: Not sure if they want their notify-IPs disclosed..

**Reply by mnordhoff** [reply]
The IPs are in their [public documentation](https://support.dnsmadeeasy.com/support/solutions/articles/47001164563-converting-secondary-zones-to-managed-dns), FWIW.

**Comment by bradley-baker** [comment]
That was exactly what I was looking for and couldn't find. Thank you so much for the help!
